dpnp/dpnp_algo/dpnp_elementwise_common.py

```diff
@@ -43,9 +44,459 @@
     "DPNPReal",
     "DPNPRound",
     "DPNPUnaryFunc",
+    "ufunc",
+    "unary_ufunc",
+    "binary_ufunc",
 ]
 
 
+import dpctl.tensor._tensor_elementwise_impl as ti
+
+import dpnp.backend.extensions.vm._vm_impl as vmi
+
+
+class ufunc:
+    """
+    ufunc()
+
+    Functions that operate element by element on whole arrays.
+
+    Calling ufuncs
+    --------------
+    op(*x[, out], **kwargs)
+
+    Apply `op` to the arguments `*x` elementwise, broadcasting the arguments.
+
+    Parameters
+    ----------
+    *x : {dpnp.ndarray, usm_ndarray}
```

Comment:
the autogenerated page is not properly describes arguments, like

> param *x : Input arrays.
type *x : {dpnp.ndarray, usm_ndarray}

while expected a list under `Parameters:`

> Parameters:     *x ({dpnp.ndarray, usm_ndarray}) -- ....

i.e. the style of description is different